examples/exercises/plot_cv_diabetes.py

```diff
@@ -60,7 +60,7 @@
 # performs cross-validation on the training data it receives).
 # We use external cross-validation to see how much the automatically obtained
 # alphas differ across different cross-validation folds.
-lasso_cv = LassoCV(alphas=alphas, cv=5, random_state=0)
+lasso_cv = LassoCV(alphas=alphas, cv=5, random_state=0, max_iter=10000)
```

Comment:
Can we check that this doesn't increase the runtime too much, and if it does, increase tolerance?

Reply:
same here.